pygeoapi/provider/postgresql.py

```diff
@@ -87,18 +90,18 @@ def __init__(self, conn_dic, table, context="query"):
         self.context = context
         self.columns = None
         self.conn = None
-        self.schema = None
 
     def __enter__(self):
         try:
-            self.schema = self.conn_dic.pop('schema', None)
-            if self.schema == 'public' or self.schema is None:
+            self.search_path = self.conn_dic.pop('search_path', None)
+            if self.search_path is ['public'] or self.search_path is None:
```

Comment:
```
>>> ['public'] is ['public']
False
```

Reply:
@alpha-beta-soup  Thank you, now I know why this is wrong (http://net-informations.com/python/iq/is.htm)

Reply:
Yes `is` and `==` can be tricky in Python 😈. Also I tend to never have `else` clauses, many refs on web like recently on [Medium](https://medium.com/better-programming/why-you-need-to-stop-using-else-statements-5b1fd09dea9e). Just a suggestion, halves number of lines, but consider this as syntactic sugar: 
```
search_path = self.conn_dic.pop('search_path', ['public'])
if search_path != ['public']:
    self.conn_dic["options"] = f'-c search_path={",".join(search_path)}'
    LOGGER.debug(f'Using explicit search path: {search_path}')
self.conn = psycopg2.connect(**self.conn_dic)
```
(IMO no need for `search_path` to be member var, in that case would also need initialization in `__init__()`))

Reply:
@justb4  Thank you for the fresh eyes, I have implemented your suggestions